### ISSUE TITLE
failed while getting Management Service public key , netbird management on k8s

### ISSUE THREAD

**Issue by tienlq2011** [netbirdio/netbird#2463]
Hello everyone, I deployed Netbird Management on Kubernetes, and when I used the Netbird client to join a peer, I encountered this error. I hope to get some help
![image](https://github.com/user-attachments/assets/3927571f-9c9b-4550-99c6-4fa31b3db9ea)

**Comment by bcmmbaga** [comment]
Hello @tienlq2011, The error you're encountering is due to an incorrect management URL. The URL is pointing to the dashboard rather than the management service.

Please review the URL you're using and ensure that it is correctly configured to point to the management service. Once you have the correct URL, you should be able to join the peer without any issues.

**Comment by tienlq2011** [comment]
![image](https://github.com/user-attachments/assets/6e737274-911d-4670-8553-b98186fb23b0)
I have checked again that the management system receives requests when I perform the join and followed the instructions on the dashboard correctly, but I'm still encountering the error as mentioned

**Comment by bcmmbaga** [comment]
The URL `https://staging-zt.lab.linksafe.vn` is currently redirecting to the dashboard instead of the management service. Could you please check your reverse proxy configuration and ensure that everything is routed correctly?

**Comment by tienlq2011** [comment]
> The URL `https://staging-zt.lab.linksafe.vn` is currently redirecting to the dashboard instead of the management service. Could you please check your reverse proxy configuration and ensure that everything is routed correctly?

Thank you, I have resolved it. I was missing the configuration to enable gRPC support on the proxy

**Comment by bcmmbaga** [comment]
Glad to hear you resolved it!. I'll go ahead and close the issue now.
